extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -635,7 +649,7 @@ private void addDiscoveredTaskToPendingCompletionTaskGroups(
     }
 
     log.info("Creating new pending completion task group for discovered task [%s]", taskId);
-    TaskGroup newTaskGroup = new TaskGroup(startingPartitions);
+    TaskGroup newTaskGroup = new TaskGroup(startingPartitions, Optional.<DateTime>absent());
```

Comment:
ah, only for tasks that are already publishing. So I guess the thought is that it won't matter here?